plugins/techdocs/src/reader/transformers/copyToClipboard.tsx

```diff
@@ -74,6 +74,31 @@ const CopyToClipboardButton = ({ text }: CopyToClipboardButtonProps) => {
   );
 };
 
+function writeToClipboard(textToCopy: string): Promise<void> {
```

Comment:
you could reuse `useCopyToClipboard` (it's already used by other plugins), so that we don't introduce/maintain an ad-hoc method

Reply:
okay